Gemfile.lock

```diff
@@ -0,0 +1,148 @@
+PATH
```

Comment:
Let's [not commit this `Gemfile.lock`](https://github.com/gonzalo-bulnes/simple_token_authentication/pull/324).